test/models/user_test.rb

```diff
@@ -87,4 +87,13 @@ class UserTest < ActiveSupport::TestCase
       users(:keith).update!(last_name: nil)
     end
   end
+
+  test "it treats blank String API keys as nil to allow for default keys" do
```

Comment:
Does this feel OK to you? I realized that clearing the value inside the form would set it to an empty string in the model. I've handled it at the model layer by returning `nil` if we have a blank value stored, which appears to be handled correctly everywhere since `blank?` or `present?` are being used where needed.

Reply:
Yes, that seems like the right solution. Good catch.